src/main/java/org/fundacionjala/sfdc/pageobjects/Accounts/SFADetailsPage.java

```diff
@@ -0,0 +1,73 @@
+package org.fundacionjala.sfdc.pageobjects.Accounts;
+
+import org.fundacionjala.sfdc.commons.DriverManager;
+import org.fundacionjala.sfdc.pageobjects.SalesForceConnection;
+import org.fundacionjala.sfdc.util.CommonActions;
+import org.openqa.selenium.Alert;
+import org.openqa.selenium.JavascriptExecutor;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'org.openqa.selenium.JavascriptExecutor'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)